website/content/en/docs/dev-guide.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Development Guide"
+linkTitle: "Development Guide"
+weight: 80
+---
+
+## Dependencies
+
+The following tools are required for doing development on Karpenter.
+
+| Package                                                            | Version  | Install                |
+| ------------------------------------------------------------------ | -------- | ---------------------- |
+| [go](https://golang.org/dl/)                                       | v1.15.3+ | `brew install go`      |
+| [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) |          | `brew install kubectl` |
+| [helm](https://helm.sh/docs/intro/install/)                        |          | `brew install helm`    |
+| Other tools                                                        |          | `make toolchain`       |
+
+## Developing
+
+### Setup / Teardown
+
+Based on which environment you are running a Kubernetes cluster, follow the [Environment specific setup](#environment-specific-setup) for setting up your environment before you continue. Once you have the environment specific settings, to install Karpenter in a Kubernetes cluster run the following commands.
+
+```
+make codegen # Create auto-generated YAML files.
+make apply # Install Karpenter
+make delete # Uninstall Karpenter
+```
+
+### Developer Loop
+* Make sure dependencies are installed
+    * Run `make codegen` to make sure yaml manifests are generated
+    * Run `make toolchain` to install cli tools for building and testing the project
+* You will need a personal development image repository (e.g. ECR)
+    * Make sure you have valid credentials to your development repository.
+    * `$KO_DOCKER_REPO` must point to your development repository
+    * Your cluster must have permissions to read from the repository
+* If you created your cluster on version 1.19 or above, you may need to tag your subnets as mentioned [here](docs/aws/README.md). This is a temporary problem with our subnet discovery system, and is being tracked [here](https://github.com/awslabs/karpenter/issues/404#issuecomment-845283904).
```

Comment:
We can remove this line as we have instructions in the getting_started doc to handle this.

Reply:
I'm not sure what exactly you are referring to here?

Reply:
In our getting started guide we give a command to tag subnets, so as long as readers follow that guide, they should be fine.

https://github.com/awslabs/karpenter/blob/main/pkg/cloudprovider/aws/docs/README.md#create-a-cluster